### ISSUE TITLE
mobile data resolution restrcition

### ISSUE THREAD

**Issue by test2a** [stom79/TubeLab#43]
### Issue

allow users to set a lower resolution on mobile network, user can manually change to higher quality but default should be like 480p to avoid lower data caps

### Steps for reproducing the issue

---

Instance:

- [ ] TubeLab
- [ ] TubeAcad

Version of TubeLab:

Android version:

**Comment by stom79** [comment]
There is an option in settings for that currently

**Comment by test2a** [comment]
Yes there is one but that is for default resolutions. Right? I was asking for a separate mobile data specific resolution setting

**Comment by stom79** [comment]
So you would like two times this option? One for mobile data and another one for WiFi, is that correct?

**Comment by test2a** [comment]
yes. that is how newpipe and almost all video apps do it. 
![Screenshot_20201107-124451](https://user-images.githubusercontent.com/39449563/98434598-3b5e9f80-20f7-11eb-915f-833c7b2c4588.png)
